src/lib/sys/configuration.cpp

```diff
@@ -337,6 +337,15 @@ uint8_t *Configuration::asCborMap(size_t &buffer_size) {
           err = cbor_encode_byte_string(&map, tmpB, tmpBSize);
         }
         break;
+      case ARRAY:
+        tmpBSize = MAX_STR_LEN_BYTES;
+        memset(tmpB, 0, MAX_STR_LEN_BYTES);
+        internalSuccess = getConfigCbor(key.keyBuffer, key.keyLen, tmpB, tmpBSize);
+        if (internalSuccess) {
+            memcpy(map.data.ptr, tmpB, tmpBSize);
+            map.data.ptr += tmpBSize;
```

Comment:
@russelldeguzman I wanted you to see this — figured out last night how to copy an already-encoded container in place without parsing while keeping the encoder state valid — it's really just [these two lines](https://github.com/sofarocean/tinycbor/blob/3cba6b11aaa0f6f674cd56ebaa573c4b65f71ee7/src/cborencoder.c#L298-L299), adapted from `append_to_buffer` in cborencoder.c!

Reply:
Ah this is great!!